cmd/gen-go-sample/gapic.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+type GapicConfig struct {
+	Interfaces []GapicInterface
+}
+
+type GapicInterface struct {
+	Name    string
+	Methods []GapicMethod
+}
+
+type GapicMethod struct {
+	Name            string
+	SampleValueSets []GapicValueSet `yaml:"sample_value_sets"`
+	Samples         struct {
+		Standalone []GapicSample
+	}
+}
+
+type GapicValueSet struct {
```

Comment:
Here and below, you're missing a few fields. I know some of them you're not implementing in this PR yet, so maybe add a `TODO`. Most importantly, the region tag in `GapicSample`, as well as the `CallingForms []string`